pkgs/development/libraries/science/chemistry/openmm/default.nix

```diff
@@ -0,0 +1,90 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, gfortran
+, fftwSinglePrec
+, doxygen
+, swig
+, python3Packages, enablePython ? false
+, opencl-headers, ocl-icd, enableOpencl ? false
+, clang, enableClang ? true
+, cudatoolkit, enableCuda ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openmm";
+  version = "7.7.0";
+
+  src = fetchFromGitHub {
+    owner = "openmm";
+    repo = pname;
+    rev = version;
+    hash = "sha256-2PYUGTMVQ5qVDeeABrwR45U3JIgo2xMXKlD6da7y3Dw=";
+  };
+
+  # "This test is stochastic and may occassionally fail". It does.
+  postPatch = ''
+    rm \
+      platforms/*/tests/Test*BrownianIntegrator.* \
+      serialization/tests/TestSerializeIntegrator.cpp
```

Comment:
and another one...
```
exception: Assertion failure at TestLangevinIntegrator.h:123.  Expected 9.97736, found 10.6743 (This test is stochastic and may occasionally fail)
```

Reply:
Meh, that is uncool. Probably half of the test suite could have stochastic failures. This is somehow how molecular dynamics works. I have a hard time figuring out which of them are expected to stochastically fail, though. Let's hope this was the last one.